### ISSUE TITLE
add schema "research project"

### ISSUE THREAD

**Issue by lkuiucsb** [lter/LTER-core-metabase#42]
The schema "research_project" will add funding code and the content required for "project" tree. The funding code is essential for EML.

**Comment by atn38** [comment]
+1, since having this schema will eliminate most of the need to use a boilerplate XML file.

**Comment by gastil** [comment]
These nine items are currently in boilerplate, at least in the MCR version of metabase: 
- access
- contact
- distribution
- funding
- intellectualRights
- language
- project
- publisher
- scope

Adding the research_project schema will take care of two of those, leaving only seven other boilerplate items to store.

**Comment by atn38** [comment]
Can we also incorporate the remaining seven into the research project
schema?

**Comment by gastil** [comment]
No. If the only piece of research_project to be used is the project tree of dataset eml, which is essentially a boilerplate item, then it would be better to have a schema called boilerplate for that and the other boilerplate items. Or you could put those into the schema for EML writing. Although I hesitate to do that for a different reason. I did put my boilerplate table into my mb2eml schema. I was not thinking ahead to the day when there would also be an mb2eml_r.
